apis/apiextensions/v1alpha1/zz_generated.usage_types.go

```diff
@@ -104,21 +107,11 @@ type Usage struct {
 	Status UsageStatus `json:"status,omitempty"`
 }
 
-// SetConditions delegates to Status.SetConditions.
```

Comment:
Merge mistake I think, these need to stay to be condition managed

Reply:
It moved into a different file per your comment below (this change in the v2 branch predates your change in main for this one type).